test/dns_nsupdate/dns_nsupdate_test.rb

```diff
@@ -1,8 +1,6 @@
 require 'test_helper'
-require 'dns/dns'
-require 'dns_nsupdate/dns_nsupdate_plugin'
+require 'dns_common/dns_common'
 require 'dns_nsupdate/dns_nsupdate_main'
-require 'dns_nsupdate/dns_nsupdate_gss_main'
```

Comment:
I've had the same problem since I don't have kerberos installed. In case you bundle install --without krb5.

I've thought about a rescue block. Then the tests fail, but at least the others will run. The alternative would be to split the gss tests into a separate file so you can test nsupdate without nsupdate_gss.